specification/intro.adoc

```diff
@@ -1,4 +1,86 @@
+:imagesdir: ./images
+
 [[intro]]
 
 == Introduction
 
+The RISC-V Confidential VM Extension <<CoVE>> specification provides
+application and virtualized workloads with data confidentiality and integrity,
+addressing one of the major datacenter security challenge. By building a
+miminized Trusted Computing Base (TCB), the CoVE interfaces manage to isolate
+workloads from each others but also from untrusted domain software components
+like the host hypervisor. CoVE implementations create a
+Confidential Computing <<CC>> framework that allows for mitigating the risks that
+guest owners are exposed to when running their workloads on shared
+infrastructures like e.g. public clouds.
+
+The value of such hardarwe-based Trusted Execution Environments (TEEs) is
+acknowledged across the industry, and Confidential Computing providers are
+looking for ways to improve the technology's scalability and performance. With
+the growing need for securing data processing, expanding Confidential Computing
+guests TCBs with trusted devices is of particular interest. Data analytics and
+transformation, artificial intelligence, financial transactions processing are
+only a few examples of confidential workloads for which a secure and performant
+I/O architecture is key to their operations. By extending guests TCBs with
+trusted accelerators, NICs or GPUs, such workloads would fully take advantage of
+their infrastructure providers capacities while keeping their data protected.
+
+The CoVE interfaces provide TEE Virtual Machines (TVMs) memory with confidential
+attributes and allow TVM guests to share parts of their address space with an
+untrusted domain. Although this enables confidential guests to be assigned with
+devices, directly or not, through para-virtualized I/O implementation, it comes
+with a significant performance cost. Withouth additional protection, TVMs have
+no ways to trust host-assigned devices and must exclude them from their TCB by
```

Comment:
I think we need to define the term "host" clearly in the glossary, because there might be confusing.

* In some context, "host" means the hypervisor / VMM. E.g. "host-assigned device", "the host hypervisor"
* In some context, "host" means the whole system or platform, including hardware. E.g. "the host platform and the assigned devices with new protection mechanisms."

Reply:
I populated the glossary section, and tried to clear that confusion.